### DISCUSSION TITLE
help adjusting firefox/display resolution settings so tar_visnetwork makes sense

### DISCUSSION THREAD

**Discussion by bbolker** [ropensci/targets#927] [Help] [answered]
I realize that this problem may lie outside the scope of `targets`, but ... the screenshot below is my Firefox window when I have zoomed the output of `tar_visnetwork()` in to a readable level. It seems as though some part of the graphics pipeline thinks that my Firefox window is only as large as its upper left corner.  This is with visNetwork 2.1.0.

![image](https://user-images.githubusercontent.com/78918/189765009-1dcab8e6-f35c-4cef-ae62-039d6626c007.png)

I have an ugly workaround:

```r
tar_visnetwork() |> visOptions(width = "200%", height = "200%")
```

![image](https://user-images.githubusercontent.com/78918/189765822-391e6be9-a3e4-488e-adb7-c6102473227c.png)

Any idea what's going on here/how to fix the settings nicely? This is Pop!OS 22.04,  Gnome 42.4, X11, 2560x1440 screen resolution, scale 100% ...

**Comment by wlandau** [comment, marked as answer]
Unfortunately, `visOptions()` is the best I can recommend. If the `visNetwork` examples at http://datastorm-open.github.io/visNetwork/ render the same, then I think it's a `visNetwork` question.
